lab-justin/lib/client-pool.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+var uuid = require('node-uuid');
+
+function registerClient(socket, clientPool) {
+  socket.wack = {};
+  socket.wack.id = 'client_' + Date.now();
+  clientPool.pool[socket.wack.id] = socket;
+}
+
+function registerClientListeners(socket, clientPool){
+  socket.on('data', function(data){
+    console.log(data.toString().toUpperCase());
+    clientPool.emit('broadcast', data);
```

Comment:
How could you modify this line so that it would include the nickname of the user who wrote the message?